build-system/tasks/create-module-compatible-scripts/index.js

```diff
@@ -0,0 +1,77 @@
+/**
+ * Copyright 2018 The AMP HTML Authors. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS-IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+const $$ = require('gulp-load-plugins')();
+const colors = require('ansi-colors');
+const gulp = $$.help(require('gulp'));
+const log = require('fancy-log');
+const MagicString = require('magic-string');
+const parser = require('@babel/parser');
+const through = require('through2');
+const traverse = require('@babel/traverse').default;
+
+
+function transform(file, encoding, callback) {
+  const code = file.contents.toString('utf8');
+  const ast = parser.parse(code);
```

Comment:
Let's verify that the parser doesn't need additional configuration when called given the input may be es2015+ syntax.

Reply:
I cant think of any options listed [here](https://babeljs.io/docs/en/next/babel-parser.html#options) to be really necessary, but would like @jridgewell to take another look